src/styles/index.scss

```diff
@@ -1,3 +1,206 @@
+@import 'resetsettings';
+@import 'variables';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
You have already set this value for the entire page, and the ``body`` tag does not need to be duplicated here.
```suggestion
```